src-tauri/src/mod_management/mod.rs

```diff
@@ -53,6 +52,35 @@ pub struct ModJson {
     version: Option<String>,
 }
 
+#[derive(Debug)]
+pub struct TempFile(fs::File, PathBuf);
```

Comment:
Just so that I understand the code correctly. The point of this struct is that when the variable is dropped, the destructor is called which deletes the file again, right?

If that's that case maybe add a comment mentioning the purpose/use-case of this struct ^^

Reply:
isn't it obvious -.-

Reply:
Took me a sec, I'm a Rust noob after all, so it would be appreciated <3

Reply:
not needed imo

Reply:
Push perms go brrrrrr 0f636df394051f777bf06ed16978963a1212862c